websites/M/Memrise/presence.ts

```diff
@@ -0,0 +1,254 @@
+const presence = new Presence({ clientId: "1095377958241304586" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/emLcjZT.png",
+	Searching = "https://i.imgur.com/OIgfjTG.png",
```

Comment:
If possible, make this one 512x512px too.

Reply:
it's already 512x512 or higher